README.md

```diff
@@ -125,7 +125,7 @@ cmake -B build
 You can pass additional options to CMake to customize the build.
 Important options are
 
-- `-G Ninja` to use the Ninja backend instead of the default Make backend. Other build backends are available with a similar syntax.
+- `-G Ninja` to use the Ninja backend instead of the default Make backend. This makes it easy to specify compiler flags for the build. Other build backends are available with a similar syntax.
```

Comment:
This is incorrect, specifying compiler flags is independent of the build backend used.

Reply:
Thanks, I'll fix this. 

At some point, I tried recompiling stdlib while specifying FFLAGS. However it didn't work until I used Ninja. 

But perhaps my problem was that I didn't delete the "build" directory (and perhaps also my CMAKE_INSTALL_PREFIX directory). 

Is it necessary to delete some build directories in that kind of situation? If yes I would like to add some statement about that, within this pull request.

Reply:
The important file in your build directory is `CMakeCache.txt`. It contains the state of the complete build directory. By deleting this file you force a reconfiguration, you could also, if you know what you are doing, modify variables in this cache directly. Deleting the complete build directory will always be the safest bet.

Reply:
Thanks -- I added a statement about deleting the build directory.